cfgov/v1/models/learn_page.py

```diff
@@ -159,143 +157,6 @@ class DocumentDetailPage(AbstractFilterPage):
     ]
 
 
-class EnforcementActionDisposition(models.Model):
```

Comment:
Note to self (for now): We need to figure out what happens between `manage.py migrate` and `runscript add_final_enforcement_data`